### PR TITLE
♻️refactor: split `profileForm` function and sort css

### PR DESCRIPTION
**Pull request by Marukome0743**
# Summary

- [x] Delete an unnecessary `useEffect`
- [x] Divide `profileForm` function into separate functions in order to clarify the related fields and methods.
- [x] Sorting css in the className
- [x] `image` type has only `FileList` type.

## How to verify your codes

I updated the test codes.
Then I tested.

### CODE REVIEW

File: components/profileForm.tsx
Comment (kazutan1230) [inline components/profileForm.tsx:1]: Have you checked whether the key "image" in "_const onSubmit_" works, when the button 「確認画面へ」 was clicked, there is correctly a selected file in the "_data_" or not,
in addition when the button 「アップロードキャンセル」 was clicked, the file is correctly deleted?

File: app/download/page.tsx
Comment (kazutan1230) [inline app/download/page.tsx:1]: I understand that most of the changes here are just reordering the styles in "_className_".
Can you explain which rules you follow and how you made this change?

File: app/page.test.tsx
Comment (kazutan1230) [inline app/page.test.tsx:1]: I understand that most of the changes here are just reordering the styles in "_className_".
Can you explain which rules you follow and how you made this change?

File: app/page.tsx
Comment (kazutan1230) [inline app/page.tsx:1]: I understand that most of the changes here are just reordering the styles in "_className_".
Can you explain which rules you follow and how you made this change?

File: app/upload/page.tsx
Comment (kazutan1230) [inline app/upload/page.tsx:1]: I understand that most of the changes here are just reordering the styles in "_className_".
Can you explain which rules you follow and how you made this change?

File: components/profileForm.tsx
Comment (kazutan1230) [reply components/profileForm.tsx:1]: I understand that most of the changes here are just reordering the styles in "_className_".
Can you explain which rules you follow and how you made this change?

**Review by kazutan1230 — COMMENTED** [review]
I understand that most of the changes here are just reordering the styles in "className".
Can you explain which rules you follow and how you made this change?

File: components/profileForm.tsx
Comment (Marukome0743) [reply components/profileForm.tsx:1]: Yup.
The image is deleted when click the cancel button.
You can confirm the useState with [React Developer Tools](https://ja.react.dev/learn/react-developer-tools).

**Comment by Marukome0743** [comment]
> I understand that most of the changes here are just reordering the styles in "className". Can you explain which rules you follow and how you made this change?

I use [useSortedClasses](https://biomejs.dev/linter/rules/use-sorted-classes/) on the new [Biome](https://biomejs.dev/) feature at [v1.6.0](https://biomejs.dev/internals/changelog/#160-2024-04-08).

File: components/profileForm.tsx
Comment (kazutan1230) [reply components/profileForm.tsx:1]: OK, I understood you checked the cancel button case.
How about
> Have you checked whether the key "image" in "_const onSubmit_" works, when the button 「確認画面へ」 was clicked, there is correctly a selected file in the "_data_"
?

> You can confirm the useState with [React Developer Tools](https://ja.react.dev/learn/react-developer-tools).

I tried React Developer Tools plugin on the Edge, and I have felt an atmosphere which the "_const onSubmit_" was going well, but I couldn't be sure.

**Review by kazutan1230 — APPROVED** [review]
I approve you to merge this PR at your own responsibility.

However, I will not responsible for whether images and  the other information can be sent properly when sending "_Profile_" using the API in the future, so please consider this or code the sending function your self.

**Pull request merged**